types/gestalt/index.d.ts

```diff
@@ -0,0 +1,1642 @@
+// Type definitions for gestalt 0.75
+// Project: https://pinterest.github.io/gestalt
```

Comment:
I believe it is better to use the code repository url(https://github.com/pinterest/gestalt ) rather than the document url